examples/mui-base-with-tailwind-css/README.md

```diff
@@ -1,47 +1,27 @@
-# Getting Started with Create React App
+# Create React App example with @mui/base & Tailwind CSS
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
-It shows how you can use @mui/base together with Tailwind CSS to build your application.
+## How to use
 
-## Available Scripts
+Download the example [or clone the repo](https://github.com/mui/material-ui):
 
-In the project directory, you can run:
+<!-- #default-branch-switch -->
 
-### `npm start`
+```sh
+curl https://codeload.github.com/mui/material-ui/tar.gz/master | tar -xz --strip=2 material-ui-master/examples/mui-base-with-tailwind-css
+cd mui-base-with-tailwind-css
+```
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+Install it and run:
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+```sh
+npm install
+npm start
+```
 
-### `npm test`
+or:
 
-Launches the test runner in interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+<!-- Tailwind CSS requires running command in the terminal, so it does not work in Codesandbox/Stackblitz with the default create react app templates. Because of this in the Codesandbox & Stackblitz links below, it is added via CDN. We will need to update this link when we do updates in the example. -->
 
-### `npm run build`
+[![Edit on StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/edit/github-bsyjuh?file=public%2Findex.html)
```

Comment:
I think that we should consider:
```suggestion
[![Edit on StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/github/mui/material-ui/tree/master/examples/mui-base-with-tailwind-css)
```

so we don't need to maintain this custom Stackblitz (that only Marija can edit without changing the URL). Also, it doesn't seem to work, how about we add `<script src="https://cdn.tailwindcss.com"></script>` in the example?

Reply:
I have added `<script src="https://cdn.tailwindcss.com"></script>` in the custom Stackblitz & Codesandbox, this is why I created a custom ones in the first place. If we add this it in the examples we won't need custom links, but we won't be really following Tailwind's guide for [create-react-app](https://tailwindcss.com/docs/guides/create-react-app). Probably it's fine considering the alternative we have now (keeping in sync these links).